coala_quickstart/coala_quickstart.py

```diff
@@ -90,15 +91,24 @@ def main():
 
     extracted_information = collect_info(project_dir)
 
-    relevant_bears = filter_relevant_bears(
-        used_languages, printer, arg_parser, extracted_information)
-
-    print_relevant_bears(printer, relevant_bears)
+    # OrderedDict used for print_relevant_bears to first print unusable bears
+    relevant_bears = OrderedDict(
+            [('unusable', {}),
```

Comment:
Not sure if I full understand this, where are we populating the dictionary with key `unusable` :sweat_smile: